FormsExtension/index.html

```diff
@@ -0,0 +1,56 @@
+     <script>
+         window.dataLayer = window.dataLayer || [];
+         function gtag(){dataLayer.push(arguments);}
+         gtag('js', new Date());
+         gtag('config', 'G-L4MQFNK7VK');
+   </script>
+<body>
+  	<div><p>Datalogics offers the Forms Extension, a software module that allows applications built using the Adobe PDF Library to work with PDF AcroForm and XFA forms documents. The extension works with the Windows 32 and 64-bit Adobe PDF Library platforms and is available for separate purchase.</p>
+<h3>What you get when you buy Forms Extension</h3>
+<ul>
+<li>A Windows supplement for working with PDF forms documents that seamlessly integrates into the Adobe PDF Library</li>
+<li>Compatibility with both AcroForm and XFA PDF forms documents</li>
+<li>The ability to convert XFA to AcroForm, import data to and export data from PDF forms documents, and flatten PDF form documents into regular PDF files</li>
+<li>Technical support from our team of digital document specialists and professionals.  You can contact your <a href="mailto:tech_support@datalogics.com?subject=Forms%20Extension">Datalogics Support representative</a> directly by electronic mail or visit our <a href="https://www.datalogics.com/resources/support/">support site</a> for this product.</li>
+<li>Regular software updates</li>
+<li>Comprehensive user documentation, provided as a PDF file and online at our Developer Resource site.</li>
+</ul>
+<h3>Working with forms technology</h3>
+<p>Adobe Inc. developed Adobe Acrobat Forms, or AcroForm, in 1998.  AcroForm is the original PDF forms technology and it remains the primary method for building PDF forms documents today. Many PDF users use Adobe Acrobat to build electronic forms and surveys, adding text fields, check boxes, radio buttons, bar codes, and other form field elements to a PDF document.</p>
+<p>XFA, or XML Forms Architecture, is a set of proprietary XML specifications for use with web forms.  In 2002 Adobe bought the company that created the XFA standard. XFA can be created and managed using the Adobe Experience Manager, previously known as Adobe LiveCycle Designer software. XFA forms can be saved internally in PDF files.  Unlike AcroForm, XFA forms can allow text reflow, so that text fields can resize to allow more content if needed. Dynamic XFA forms can also be interactive, as the form document is built using a series of templates. The form document can respond to the information that a user enters or selects and display a different set of form fields relevant to the user’s answer.</p>
+<p>Many XFA documents remain in use, but most PDF documents use AcroForm instead. AcroForm are compatible with a much wider range of software applications, as well as with Acrobat itself. Note also that XFA was deprecated in the new PDF 2.0 format, and is not permitted for use with special ISO formats for PDF documents, including PDF/A, PDF/E, PDF/X, PDF/UA, and PDF/VT. Further, XFA is not supported on mobile devices.</p>
```

Comment:
Im not sure, but would it be better to have "AcroForms are compatible" or "AcroForm is compatible" instead?